runtime/src/bank_forks.rs

```diff
@@ -649,6 +649,21 @@ impl ForkGraph for BankForks {
     }
 }
 
+impl Drop for BankForks {
+    fn drop(&mut self) {
+        info!("BankForks::drop(): started...");
+        // It's okay to abruptly drop all banks here albeit BankForks provides panic-happy Index
+        // impl on them. We're inside the very end of life of it (i.e. the Drop impl block!),
```

Comment:
Sorry, I'm confused what you mean by this comment about `BankForks` implementing `Index`. What is the relevance here?

When we're dropping we have unique access (&mut), so there's no chance that some other thread is accessing the banks via `Index`, right?

Reply:
err, just removed the comment: c5b866cde2b77dc098aaab52956f9e9c1510b9df